File: March/13-03-2024/main.js
import readline from "readline-sync";

var rows = readline.questionInt("JavaScript\nEnter the number of rows: "); 

for (var i = 1; i <= rows; i++) {
    var ans = "";

    for (var j = 1; j <= i; j++) {
        ans += "*";
    }

    for(var k = 1; k <= (rows - i) * 2; k++) {
        ans += " ";
    }

    for(var l=1; l <= i; l++) {
        ans += "*";
    }

    console.log(ans);

}

File: March/13-03-2024/main.cpp
#include <iostream>

using namespace std;

int main() {
    int rows;
    cout << "C++";
    cout << "Enter the number of rows: ";
    cin >> rows;

    for (int i = 1; i <= rows; i++) {
        for (int j = 1; j <= i; j++) {
            cout << "*";
        }

        for (int k = 1; k <= (rows - i) * 2; k++) {
            cout << " ";
        }

        for (int l = 1; l <= i; l++) {
            cout << "*";
        }


        cout << endl;
    }   


    return 0;
}